### PR TITLE
[Flutter-Parent][MBL-13663] Consolidate Error Widget

### PR DESCRIPTION
**Pull request by StephenBrough**

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1014275&projectid=20)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

File: apps/flutter_parent/lib/l10n/app_localizations.dart [hunk 1]
```diff
@@ -501,4 +502,14 @@ class AppLocalizations {
         'Filter',
         desc: 'Label for buttons to filter what items are visible',
       );
+
+  String genericLoadingErrorMessage(String loadTarget) =>
+      Intl.message(
+        'There was an error loading your $loadTarget.',
```
Comment (JordanMarshall) [inline apps/flutter_parent/lib/l10n/app_localizations.dart]: Let's use separate messages for this to better account for differences in plurals and gender.

File: apps/flutter_parent/lib/utils/common_widgets/error_panda_widget.dart [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  final Function callback;
+  final String loadTarget;
+
+  ErrorPandaWidget(this.loadTarget, this.callback);
+
+  @override
+  Widget build(BuildContext context) {
+    return LayoutBuilder(
+      builder: (context, constraints) {
+        return SingleChildScrollView(
```
Comment (JordanMarshall) [inline apps/flutter_parent/lib/utils/common_widgets/error_panda_widget.dart]: Take a look a FullScreenScrollContainer

**Pull request merged**